test/features/export/Export.spec.js

```diff
@@ -133,5 +136,20 @@ describe('modules/export - Export', () => {
                 additionalData: { foo: 'bar' },
             });
         });
+
+        it('should fire svg export', () => {
+            eventBus.on('export', (context) => {
+                const parser = new DOMParser();
+                const doc = parser.parseFromString(
+                    context.payload,
+                    context.mimeType
+                );
+
+                expect(doc.documentElement).toBeDefined();
+                expect(doc.documentElement.nodeName).toBe('svg');
```

Comment:
I dont get what this test is ensuring.

Reply:
It ensures that the behavior returns an XML document that can be parsed and has a root node named svg. There is certainly room for improvement, like making sure the document contains all elements. This is not yet checked for any of the export types.

Reply:
Yeah, also the expects are in the callbacks. This leads to the behavior that the event doesn't need to run to succeed the test, which is not what I would expect. There is missing something to ensure, that the callback is running and executing the expects inside the callback.

Reply:
You are absolutely right. Will be fixed in #101.